bliss/models/decoder.py

```diff
@@ -471,11 +464,10 @@ def __init__(self, tile_slen, ptile_slen, n_bands, galaxy_model: GalaxyModel):
             raise TypeError("galaxy_model is not a valid type.")
         self.galaxy_decoder = galaxy_decoder
 
-    def forward(self, locs, galaxy_params, galaxy_bools):
+    def forward(self, galaxy_params: Tensor, galaxy_bools: Tensor):
```

Comment:
do you think we need the separate `_render_single_galaxies` method? `forward` does very little besides calling this method

Reply:
We can probably do away with this class altogether in a separate PR